### PR TITLE
removed duplicate definitions

### PR DESCRIPTION
**Pull request by airenzp**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: server/shared/types/terms/samplelst.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { Term, TermWrapper, BaseQ } from '../termdb.ts'
+import { BaseTerm, BaseQ } from './term.ts'
+import { TermWrapper } from './tw.js'
```
Comment (creilly8) [inline server/shared/types/terms/samplelst.ts:2]: I think this is supposed to be `import { TermWrapper } from './tw'`

File: server/shared/types/terms/geneExpression.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
-import { TermWrapper, BaseQ, Term } from '../termdb.ts'
+import { TermWrapper } from './tw.ts'
+import { BaseQ } from './term.ts'
 import { TermSettingInstance } from '../termsetting.ts'
+import { NumericTerm } from './numeric.js'
```
Comment (creilly8) [inline server/shared/types/terms/geneExpression.ts:4]: I think this is supposed to be `import { NumericTerm } from './numeric'`

File: server/shared/types/terms/samplelst.ts (hunk 1 above)
Comment (airenzp) [reply server/shared/types/terms/samplelst.ts]: Thanks Colleen the extension if added should be ts, good you noticed this. In some files the imports have the extension and in some not so I follow the format adopted.

**Review by xzhou82 — APPROVED** [review]
thanks

**Pull request merged**